pbdf/Issues/drivinglicence/description.xml

```diff
@@ -0,0 +1,202 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Driving Licence</en>
+		<nl>rijbewijs</nl>
+	</Name>
+	<ShortName>
+		<en>Driving Licence</en>
+		<nl>rijbewijs</nl>
+	</ShortName>
+	<SchemeManager>pbdf-staging</SchemeManager>
+	<IssuerID>pbdf</IssuerID>
+	<CredentialID>drivinglicence</CredentialID>
+	<Description>
+		<en>Your personal details derived from your driving licence</en>
+		<nl>Je persoonsgegevens volgens je rijbewijs</nl>
+	</Description>
+	<IssueURL>
+		<en>https://passport-issuer.staging.yivi.app/en</en>
+		<nl>https://passport-issuer.staging.yivi.app/nl</nl>
+	</IssueURL>
+
+	<ForegroundColor>#ffffff</ForegroundColor>
+	<BackgroundGradientStart>#053F58</BackgroundGradientStart>
+	<BackgroundGradientEnd>#165B78</BackgroundGradientEnd>
+
+	<IsInCredentialStore>true</IsInCredentialStore>
+	<Category>
+		<en>Personal</en>
+		<nl>Persoonlijk</nl>
+	</Category>
+	<FAQIntro>
+        <en>Add your personal data from a Machine Readable driving licence to your Yivi app.</en>
+        <nl>Voeg persoonsgegevens van je rijbewijs toe aan je Yivi-app.</nl>
+    </FAQIntro>
+    <FAQPurpose>
+        <en>Your personal data in the Yivi app allows you to identify yourself online. Then you can, for example, disclose your name or prove that you are over 18 years old (without revealing your date of birth).</en>
+        <nl>Met je persoonsgegevens in de Yivi-app kun je jezelf online identificeren. Je kunt dan bijvoorbeeld je naam delen of aantonen dat je ouder bent dan 18 jaar (zonder je geboortedatum prijs te geven).</nl>
+    </FAQPurpose>
+    <FAQContent>
+        <en>You will add the following data:\n• photo,\n• document number,\n• first name(s),\n• last name,\n• issuing authority,\n• date of birth,\n• place of birth,\n• year of birth,\n• date of expiry,\n• whether you are over 12, 16, 18, 21 or 65,\n• whether active authentication was performed.</en>
+        <nl>Je voegt de volgende gegevens toe:\n• foto,\n• documentnummer,\n• voornaam/voornamen,\n• achternaam,\n• uitgevende autoriteit,\n• geboortedatum,\n• geboorteplaats,\n• geboortejaar,\n• vervaldatum,\n• of je ouder bent dan 12, 16, 18, 21 of 65,\n• of er actieve authenticatie heeft plaatsgevonden.</nl>
+    </FAQContent>
+    <FAQHowto>
+        <en>1. Scan your driving licence with your phone to read the Machine Readable Zone (MRZ).\n2. Extract the data using the NFC chip in your phone.\n\nWe aim to support this functionality for EU driving licences.</en>
+        <nl>1. Scan de voorkant van je rijbewijs met je telefoon\n2. Haal de gegevens op via de NFC-chip in je telefoon.\n\nWe streven ernaar om deze functionaliteit te ondersteunen voor EU-rijbewijzen.</nl>
```

Comment:
Maybe 'scan' is not fully self-explanatory?
Maybe make it something like `1. Scan de voorkant van je rijbewijs met de camera van je telefoon ...`.
Same for English